app/Http/Controllers/Admin/ProjectController.php

```diff
@@ -176,4 +196,18 @@ public function updateImage($projectRequest, $oldImage)
 
         return $result;
     }
+
+    /**
+     * Delete the uploaded image.
+     *
+     * @param string $imageName | image name.
+     *
+     * @author Ali, Muamar
+     */
+    public function deleteImage(string $imageName)
+    {
+        if (File::exists(public_path(sprintf('images/%s', $imageName)))){
+            File::delete(public_path(sprintf('images/%s', $imageName)));
```

Comment:
i dont understand why you need dedicated controller to update create and delete images.